server/src/internalClusterTest/java/org/elasticsearch/snapshots/RepositoriesIT.java

```diff
@@ -302,7 +302,10 @@ public void testRepositoryConflict() throws Exception {
         logger.info("--> try updating the repository, should fail because the deletion of the snapshot is in progress");
         RepositoryConflictException e2 = expectThrows(
             RepositoryConflictException.class,
-            clusterAdmin().preparePutRepository(repo).setType("mock").setSettings(Settings.builder().put("location", randomRepoPath()))
+            clusterAdmin().preparePutRepository(repo)
+                .setVerify(false)
```

Comment:
will deadlock on snapshot thread pool, we are running with single thread which is busy at the moment

Reply:
Can we have this comment in the code please?